lib/tmdb/client.rb

```diff
@@ -104,116 +108,134 @@ def common_actors_between_movies(movie_one_title, movie_two_title)
       end
 
       def person_autocomplete(query)
-        data = get_parsed_multi_search_results(query)
+        url = url_for_multi_search(query)
+        data = get_data(url)&.dig(:results)
         person_results = data.select{ |result| result[:media_type] == "person"}
         person_results.map{ |result| result[:name] }.uniq
       end
 
       def person_detail_search(person_id)
-        bio_results = get_parsed_person_bio(person_id)
-        movie_credits_results = get_parsed_person_movie_credits(person_id)
-        tv_credits_results = get_parsed_person_tv_credits(person_id)
+        person_params = { person_id: person_id }
+        person_data = request(:person_data, person_params)
+
+        # person_url = url_for_person_data(person_id)
+        # person_data = get_data(person_url)
```

Comment:
@mikevallano This whole PR is super WIP. 
That said, most of this `person_detail_search` method uses the `url_for_...` convention we were talking about a couple of days ago. Then I went rogue and tried a different approach on lines 118 & 119. It corresponds with `request` method on 183 (which is just a start of the idea as well). It might be a little _too_ fancy. We'll start to see some grossness when we encounter the "discover search" url builder -- which i guess could be its own method... 🤔 

I just wanted to throw this out there as a visual aid for our discussion.